src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,122 @@
+                    Boolean restaurantOpen = Boolean.valueOf(request.queryParams("restaurantOpen"));
+                    double restaurantRating = Double.valueOf(request.queryParams("restaurantRating"));
+                    insertRestaurant(conn, restaurantName, restaurantOpen, restaurantRating);
+
+                    response.redirect("/");
+                    return "";
+                })
+        );
+
+        Spark.post(
+                "/deleteRestaurant",
+                ((request, response) ->
+                {
+                    public static void deleteRestaurant (Connection conn,int id) throws SQLException {
+                    PreparedStatement stmt = conn.prepareStatement("DELETE restaurant SET is_Open = False is_Open WHERE id = ?");
+                    stmt.setInt(1, id);
+                    stmt.execute();
+                }
+                };
+
+        Spark.get(
+                "/selectRestaurant",
+                ((request, response) -> {
+
+                    String restaurantName = request.queryParams("restaurantName");
+                    ArrayList<RestaurantList> (Connection conn) throws SQLException {
+                        ArrayList<RestaurantList> restaurants = new ArrayList<>();
+                        Statement stmt = conn.createStatement();
+                        ResultSet results = stmt.executeQuery("SELECT * FROM restaurants");
+                        while (results.next()) {
+                            int id = results.getInt("id");
+                            String text = results.getString("text");
+                            boolean isDone = results.getBoolean("is_Open");
+                            double isOpen = results.getDouble("rating");
+                            restaurants.add(new RestaurantList(id, text, isOpen, rating));
+                        }
+                        return restaurants;
+                    }
+                    )};
+
+                    Spark.post(
+                            "/UpdateRestaurant",
+                            ((request, response) ->
+                            {
+                                public static void deleteRestaurant (Connection conn,int id) throws SQLException {
+                                PreparedStatement stmt = conn.prepareStatement("DELETE restaurant SET is_Open = False is_Open WHERE id = ?");
+                                stmt.setInt(1, id);
+                                stmt.execute();
+                            }
+                            }));
+
+                    Spark.get(
+                            "/",
+                            ((request, response) -> {
+                                HashMap m = new HashMap<>();
+                                m.put(1, "Olive Garden" );
+                                return new ModelAndView(m, "home.html");
+                            },
+                            new MustacheTemplateEngine()
+                            }));
```

Comment:
If you have questions about the direction you need to go.  Ask me and we can get you back on track.